test/heatmap-border-radius.html

```diff
@@ -0,0 +1,166 @@
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+'License'); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+'AS IS' BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<html>
+
+<head>
+    <meta charset='utf-8'>
+    <script src='lib/simpleRequire.js'></script>
+    <script src='lib/config.js'></script>
+    <meta name='viewport' content='width=device-width, initial-scale=1' />
+</head>
+
+<body>
+    <style>
+        html,
+        body,
+        #main,
+        #main2 {
+            width: 100%;
+            height: 100%;
+            margin: 0;
+        }
+    </style>
+    <div id='main'></div>
+    <div id='main2'></div>
+    <script>
+
+        require([
+            'echarts'
+        ], function (echarts) {
+
+            var chart = echarts.init(document.getElementById('main'));
+            var chartTwo = echarts.init(document.getElementById('main2'));
+
+            var hours = ['12a', '1a', '2a', '3a', '4a', '5a', '6a',
```

Comment:
Try using `npm run mktest` command to create a new test case and put all heatmap borderRadius related test cases in one file.

Reply:
I had done to combining all tests demo about heatmap-border-radius.